website/docs/r/org_local_user.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_org_local_user"
+sidebar_current: "docs-vcfa-resource-org-local-user"
+description: |-
+  Provides a resource to manage local users in VMware Cloud Foundation Automation Organizations.
+---
+
+# vcfa\_org\_local\_user
+
+Provides a resource to manage local users in VMware Cloud Foundation Automation Organizations.
+
+## Example Usage
+
+```hcl
+resource "vcfa_org" "demo" {
+  name         = "terraform-org"
+  display_name = "terraform-org"
+  description  = "Terraform demo"
+  is_enabled   = true
+}
+
+data "vcfa_role" "org-admin" {
+  org_id = vcfa_org.demo.id
+  name   = "Organization Administrator"
+}
+
+resource "vcfa_org_local_user" "demo" {
+  org_id   = vcfa_org.demo.id
+  role_id  = data.vcfa_role.org-admin.id
+  username = "demo-local-user"
+  password = "CHANGE-ME"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `org_id` - (Required) An Org ID for this Local User to be created in 
+- `role_id` - (Required) A role ID to assign to this user
```

Comment:
User:role is a 1:N relationship. Looks like we're missing ability to set multiple roles.

Reply:
Changed. `role_ids` instead of `role`